external_search.py

```diff
@@ -47,12 +52,20 @@ def __init__(self, _db, url=None, works_index=None):
                     goal=ExternalIntegration.SEARCH_GOAL
                 )
 
+                if not integration:
+                    raise CannotLoadConfiguration(
+                        "No Elasticsearch integration configured."
+                    )
                 url = url or integration.url
                 if not works_index:
-                    works_index = integration.setting(self.WORKS_INDEX_KEY).value
-
+                    setting = integration.setting(self.WORKS_INDEX_KEY).value
+                    works_index = setting.value_or_default(
+                        self.DEFAULT_WORKS_INDEX
+                    )
```

Comment:
Most of the time there's no need to make the administrator explicitly define a DEFAULT_WORKS_INDEX.